pkg/aks/check.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/Azure/azure-sdk-for-go/services/operationalinsights/mgmt/2020-08-01/operationalinsights"
 	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/rancher/aks-operator/pkg/aks/services"
```

Comment:
From what I can tell, @alexander-demicev you are extracting the `*containerservices` calls and switching to use the Azure client functions defined in the new `services` package. Seems fine design wise, I just want to make sure there are no functional gaps there so both provisioned/provisioned cluster update and an imported Azure cluster should be tested